_data/email.yml

```diff
@@ -213,6 +213,7 @@ websites:
       - totp
       - hardware
```

Comment:
Let's fix the hardware tag as well while we're at it.
```suggestion
      - u2f
```

Reply:
![firefox_PFbhu3qndV](https://user-images.githubusercontent.com/3535780/72211774-3a59e300-34d1-11ea-9e3a-374514930119.png)
There should be a "Commit suggestion" button available to you. Sadly I can't use it because of a GitHub bug 🙄

Reply:
Should it have a leading space in front of the dash like " - hardware" does? By clicking "Commit changes," I am approving your additional change, right?

Reply:
The leading space is another GitHub bug hehehe 🙁 
Yes the "Commit changes" incorporates my suggestion into the PR.